gallery/src/components/NxDropdown/NxDropdownCloseHandlerExample.tsx

```diff
@@ -18,10 +18,10 @@ function NxDropdownCloseHandlerExample() {
                 onToggleCollapse={onToggleCollapse}
                 onCloseClick={(evt: MouseEvent) => evt.preventDefault()}
                 onCloseKeyDown={(evt: KeyboardEvent) => evt.preventDefault()}>
-      <a onClick={onClick} href="#/pages/Dropdown" className="nx-dropdown-button">
+      <a onClick={onClick} href="#/pages/Dropdown" role="menuitem" className="nx-dropdown-button">
         Link
       </a>
-      <button onClick={onClick} className="nx-dropdown-button">
+      <button role="menuitem" onClick={onClick} className="nx-dropdown-button">
```

Comment:
I think we should create convenience components that encapsulate the needed classes and roles

Reply:
> I think we should create convenience components that encapsulate the needed classes and roles

The tooltip logic will need to be adjusted to accomodate this as well

Reply:
> > I think we should create convenience components that encapsulate the needed classes and roles
> 
> The tooltip logic will need to be adjusted to accomodate this as well

This is still in progress, but I wanted to get your thoughts on the addition of adding wrapper prop to withClass here: 3353a939799a2e11b4606976a08be82e902252d9

Reply:
> This is still in progress, but I wanted to get your thoughts on the addition of adding wrapper prop to withClass here: https://github.com/sonatype/sonatype-react-shared-components/commit/3353a939799a2e11b4606976a08be82e902252d9

Hmm is there any other kind of wrapper that would make sense here besides `NxOverflowTooltip`? I wonder if we should just add a boolean flag for that possibility only, and leave anything more obscure for custom component implementations.

Reply:
> > This is still in progress, but I wanted to get your thoughts on the addition of adding wrapper prop to withClass here: [3353a93](https://github.com/sonatype/sonatype-react-shared-components/commit/3353a939799a2e11b4606976a08be82e902252d9)
> 
> Hmm is there any other kind of wrapper that would make sense here besides `NxOverflowTooltip`? I wonder if we should just add a boolean flag for that possibility only, and leave anything more obscure for custom component implementations.

I can't think of anything else that would be wrapper. I added a boolean flag for `NxOverflowTooltip` here: 641905f546159b0d24ca397f0c2308a99990992d